aiodns/__init__.py

```diff
@@ -271,11 +268,15 @@ def cancel(self) -> None:
     def _sock_state_cb(self, fd: int, readable: bool, writable: bool) -> None:
         if readable or writable:
             if readable:
-                self.loop.add_reader(fd, self._handle_event, fd, READ)
+                self.loop.add_reader(
+                    fd, self._channel.process_fd, fd, pycares.ARES_SOCKET_BAD
+                )
                 self._read_fds.add(fd)
             if writable:
-                self.loop.add_writer(fd, self._handle_event, fd, WRITE)
-                self._write_fds.add(fd)
+                self.loop.add_writer(
+                    fd, self._channel.process_fd, pycares.ARES_SOCKET_BAD, fd
+                )  # pragma: no cover
+                self._write_fds.add(fd)  # pragma: no cover
```

Comment:
Why are there no cover comments added here?

Reply:
Yeah, it's definitely unacceptable to drop test coverage.